Prac_06/guitars.py

```diff
@@ -0,0 +1,22 @@
+from Prac_06.guitar import Guitar
+
+guitars = []
+
```

Comment:
Using a function for your code can make it neater:
`def main()
(code in here)
main()`